pkg/contracts/src/CVStrategy.sol

```diff
@@ -742,7 +690,7 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
             proposal.voterStakedPointsPct[_sender] = stakedPointsPct;
 
             // console.log("_sender", _sender);
-            uint256 stakedAmount = convertPctToTokens(stakedPointsPct);
+            uint256 stakedAmount = stakedPointsPct;
```

Comment:
could we also get rid of that stakedPointsPct?

Reply:
yeah i want give big refactor after we get everything merged. i can test doing just that change names for that branch at least.

Reply:
renamed